core/build.gradle

```diff
@@ -2,17 +2,17 @@ apply plugin: 'com.android.library'
 apply plugin: 'bintray-release'
 
 dependencies {
-    compile 'com.android.support:support-v4:19.1.0'
+    compile 'com.android.support:support-v4:23.0.1'
 }
 
 android {
     defaultConfig {
         minSdkVersion 14
-        targetSdkVersion 19
+        targetSdkVersion 22
```

Comment:
But to be clear, whether this value is 22 or 23, we're at risk when we use Merlin in a project that is running on M? i.e. we _need_ to handle it right?